python/lsst/obs/lsstSim/lsstSimMapper.py

```diff
@@ -305,6 +312,12 @@ def bypass_deepCoaddId(self, datasetType, pythonType, location, dataId):
     def bypass_deepCoaddId_bits(self, datasetType, pythonType, location, dataId):
         return 1 + 7 + 13*2 + 3
 
+    def bypass_dcrCoaddId(self, datasetType, pythonType, location, dataId):
+        return self._computeCoaddExposureId(dataId, True)
+
+    def bypass_dcrCoaddId_bits(self, datasetType, pythonType, location, dataId):
+        return 1 + 7 + 13*2 + 3
```

Comment:
Since this is copied from `bypass_deepCoaddId_bits`, should it just call that instead?